Src/Support/IntegrationTests/DataWrapperTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿/*
+Copyright 2017 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Services;
+using Google.Apis.Translate.v2;
+using Google.Apis.Translate.v2.Data;
+using IntegrationTests.Utils;
+using System.Collections.Generic;
+using Xunit;
+
+namespace IntegrationTests
+{
+    public class DataWrapperTests
+    {
+        [Fact]
+        public void Translate()
```

Comment:
Could you add another test for Detect? That's the only other operation that uses a body within Translate, as far as I can tell.

Reply:
Done